x-pack/plugins/session_view/public/components/DetailPanelAccordion/styles.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useMemo } from 'react';
+import { CSSObject } from '@emotion/react';
+
+export const useStyles = () => {
+  const cached = useMemo(() => {
+    const tabSection: CSSObject = {
+      padding: '16px',
```

Comment:
For example, after importing the hook like so:

```tsx
const { euiTheme } = useEuiTheme();
```

You can then use all the variables available:
```suggestion
      padding: euiTheme.size.base,
```

Reply:
ah yep, i think for the most part we've tried to use euiTheme for these things, but looks like there are some spots missing. Thanks for the review!

Reply:
👍  A good way to quickly check on your usage is just to test in dark mode (Advanced Settings -> search `dark`). Also I did find another ~40 instances of `px`. Probably worth just going back through all the `styles.ts` files.

Reply:
Thanks for pointing all these euiTheme properties @cchaos , we will update and take a look across all styling files.

Question, is it a norm for designers to follow pre-determined EUI Sizes for all spacing? For example, we currently have a 28px left margin spacing, should that be changed to the nearest size (`size.l` : 24 or `size.xl` : 32), or it's fine to have this kind of exception?

Reply:
I would confirm with your designer where very specific values are needed or if the idea is just to use general spacing in which we recommend sticking with the variables so everything scales the same.